source/include/ota.h

```diff
@@ -319,6 +319,70 @@ typedef struct OtaAgentContext
  * @return OtaErr_t The state of the OTA Agent upon return from the OtaState_t enum.
  * If the agent was successfully initialized and ready to operate, the state will be
  * OtaAgentStateReady. Otherwise, it will be one of the other OtaState_t enum values.
+ *
+ * <b>Example</b>
+ * @code{c}
+ * // Application callback when the OTA agent has completed the job
```

Comment:
We have more events now in the app callback than the two mentioned

Reply:
Added a note to reflect this.